src/Laserfiche.Repository.Api.Client.csproj

```diff
@@ -2,6 +2,17 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <Version>1.0.1-beta.1</Version>
+    <Authors>Laserfiche</Authors>
+    <Description>.NET API that enables easy and secure access to Laserfiche Repository.</Description>
+    <Copyright>Copyright 2022 Laserfiche</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/Laserfiche/lf-repository-api-client-dotnet</PackageProjectUrl>
+    <PackageIcon>logo.png</PackageIcon>
+    <PackageIconUrl />
+    <RepositoryUrl>https://github.com/Laserfiche/lf-repository-api-client-dotnet</RepositoryUrl>
+    <RepositoryType>GitHub</RepositoryType>
+    <PackageTags>Laserfiche Repository Data Document-Management Document-Store</PackageTags>
```

Comment:
Are these package tags relevant to the repo client lib? 
Data Document-Management Document-Store

Reply:
I guess so? I can remove them but it's similar how we have tags for oauth client lib.

Reply:
Just to add reasoning for the previous comment: The repp client lib connects to Laserfiche Repository which does store data and document and to some degree it allows document management.